packages/hash/api/src/seed-data/index.ts

```diff
@@ -1,82 +1,31 @@
 import { Logger } from "@hashintel/hash-backend-utils/logger";
-import {
-  SYSTEM_ACCOUNT_NAME,
-  SYSTEM_ACCOUNT_SHORTNAME,
-} from "@hashintel/hash-shared/environment";
 import { GraphApi } from "@hashintel/hash-graph-client";
-import { OrgModel, OrgSize } from "../model";
-import { systemAccountId } from "../model/util";
 import { ensureUsersAreSeeded } from "./seed-users";
 import { PageDefinition, seedPages } from "./seed-pages";
+import { OrgModel } from "../model";
 
-// Seed Org with some pages.
-const seedOrg = async (params: {
+export const seedUsers = async (params: {
   graphApi: GraphApi;
   logger: Logger;
-}): Promise<OrgModel> => {
-  const { graphApi, logger } = params;
-
-  const existingOrgModel = await OrgModel.getOrgByShortname(graphApi, {
-    shortname: SYSTEM_ACCOUNT_SHORTNAME,
-  });
-
-  if (existingOrgModel) {
-    return existingOrgModel;
-  }
+  orgModel: OrgModel;
+}): Promise<void> => {
+  const { graphApi, logger, orgModel } = params;
 
-  const sharedOrgModel = await OrgModel.createOrg(graphApi, {
-    name: SYSTEM_ACCOUNT_NAME,
-    shortname: SYSTEM_ACCOUNT_SHORTNAME,
-    providedInfo: {
-      orgSize: OrgSize.ElevenToFifty,
-    },
-    actorId: systemAccountId,
+  const createdUsers = await ensureUsersAreSeeded({
+    ...params,
+    systemOrgAccountId: orgModel.getEntityUuid(),
```

Comment:
This is assuming you're passing in the system org. Please change this.

Reply:
This no longer makes such an assumption after the changes in:
- https://github.com/hashintel/hash/pull/1596/commits/2101b462063712aa6e9ab795f2db22ec59236864
- https://github.com/hashintel/hash/pull/1596/commits/7634f6395ff669efabfaed4fc34c28a02aceea86